connection.go

```
@@ -303,6 +308,29 @@ func (sc *snowflakeConn) ExecContext(ctx context.Context, query string, args []d
 }
 
 func (sc *snowflakeConn) QueryContext(ctx context.Context, query string, args []driver.NamedValue) (driver.Rows, error) {
+	qid, _ := getResumeQueryID(ctx)
```

`_` err is being dropped, we can check it